activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+*   The deferrable foreign key can be passed to `t.references`.
+
+    Fixed a bug in which deferrable foreign_key was ignored when passed to `t.references`.
```

Comment:
Let's remove this line. Just saying "The deferrable foreign key can be passed to `t.references`." should be enough.